mucommander-translator/src/main/resources/dictionary_ko.properties

```diff
@@ -317,7 +322,6 @@ status_bar.volume_capacity = 용량: {0}
 shortcuts_panel.title = 바로가기
 shortcuts_panel.restore_defaults = 기본값으로 복원
 shortcuts_panel.show = 표시
-shortcuts_panel.filter=필터
```

Comment:
you removed it because it's missing in dictionary.properties? it shouldn't be removed - it's used in the Preferences -> Shortcuts tab